packages/components/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -107,6 +106,11 @@ interface BaseConfirmationModalProps {
    */
   readonly cancelLabel?: string;
 
+  /**
+   * Action Type (Work or destructive) for confirm button.
+   */
+  readonly type?: "work" | "destructive" | "learning" | undefined;
```

Comment:
I wouldn't expect there to be a learning variation of the ConfirmationModal.. I'm not sure if we explicitly need to say you _can't_ use `learning`, but I would _prefer_ it if we can restrict it to just work and destructive for now.

Reply:
the type is passed down to the variation of button type, to keep it simple I kept the learning option.  🤔

Reply:
I think we would only want the 2 options available. If learning is not available on here, that means it can not be passed down (which is what we want.

also, by adding the `?` in `type?:` you are saying that this prop is optional. Means that you do not need to also allow for `undefined`